spec/requests/providers_spec.rb

```diff
@@ -1875,6 +1887,16 @@ class DummyProvider; end
 
         expect(response).to have_http_status(:forbidden)
       end
+
+      it "returns a 404 for a provider that does not support configured_systems" do
```

Comment:
minor, maybe update this and the PR description.

Reply:
Definitely, I meant to update the test case descriptions and forgot. Though I updated the PR description but I must have forgotten to save. I'll take care of both. Thanks!